exampleCourse/questions/gradientVectorFunction/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "uuid": "acff9a90-0145-486a-87b0-65200c4e0253",
+    "title": "Obtain the gradient of a vector",
```

Comment:
Can we change this question to be named and formatted as an example, like `examplesString`, and put it in the `plelementexamples` homework?

Also add the question to the list for automatic testing in `tests/testQuestions.js`.